packages/smooth_app/lib/pages/product/edit_ingredients_page.dart

```diff
@@ -178,7 +178,26 @@ class _EditOcrPageState extends State<EditOcrPage> {
           ),
         );
       } else {
-        children.add(Container(color: Colors.white));
+        children.add(
+          Container(
+            alignment: Alignment.center,
+            margin: EdgeInsets.only(bottom: size.height * 0.25),
+            child: Column(
+              mainAxisAlignment: MainAxisAlignment.center,
+              crossAxisAlignment: CrossAxisAlignment.center,
+              children: <Widget>[
+                const Icon(
+                  Icons.image_not_supported,
+                  size: 200,
```

Comment:
Where does that `200` come from?
If I see your screenshots, there seems to be already a space / positioning in place (on both pictures the refresh/upload photo button is at the same place).
I suggest that you just fill the space (`Expand`?) with the icon instead of specifying a fixed size of `200`.
That said, if `200` already appears somewhere else in the code (for positioning purposes), please make it a `static const`.
(not fan of the margin either, for the same reasons: why `0.25`?)

Reply:
Hello @monsieurtanuki, 
Thanks for the review. I changed most of the things you asked for.
I removed the 200, instead used the size object.height / 4. And also for 0.25 I used the same size object. Should be consistent for all screen size.